doc/association-rules.rst

```diff
@@ -0,0 +1,79 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+#### Inputs
+
+-  **Data**
+
+Data set
+
+#### Outputs
+
+-  **Matching Data**
+
+Data instances matching the criteria.
+
+Description
+-----------
+
+This widget runs the original `Agrawal's algorithm <http://cs-people.bu.edu/evimaria/cs565/advances.pdf>`__ for the induction of
```

Comment:
Sorry, but this is wrong. See the first sentence: https://orange3-associate.readthedocs.io/en/latest/